.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -28,7 +38,7 @@ If you are not running the latest version of Terraform, please try upgrading bec
 <!--
 Paste the relevant parts of your Terraform configuration between the ``` marks below.
 
-For large Terraform configs, please use a service like Dropbox and share a link to the ZIP file. For security, you can also encrypt the files using our GPG public key.
+For Terraform configs larger than a few resources, or that involve multiple files, please make a GitHub repository that we can clone, rather than copy-pasting multiple files in here. For security, you can also encrypt the files using our GPG public key.
```

Comment:
As a user entering a bug, where do I find the GPG public key ?

Reply:
Good question! I just added a link to it.